extensions/wikia/Chat2/Chat.class.php

```diff
@@ -87,7 +87,9 @@ public static function getSessionKey() {
 	 *
 	 * @return true|string Returns true on success, returns an error message as a string on failure.
 	 */
-	public static function banUser( $subjectUserName, User $adminUser, $time, $reason ) {
+	public static function banUser( string $subjectUserName, User $adminUser, int $time, string $reason ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce the number of returns of this function 4, down to the maximum allowed 3. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1142)